elide-contrib/elide-swagger/src/main/java/com/yahoo/elide/contrib/swagger/SwaggerBuilder.java

```diff
@@ -509,19 +510,21 @@ private Parameter getSortParameter() {
                         .description("Filters the collection of " + typeName + " using a 'joined' RSQL expression"));
             }
 
-            for (Operator op : filterOperators) {
-                attributeNames.forEach((name) -> {
-                    Class<?> attributeClass = dictionary.getType(type, name);
-
-                    /* Only filter attributes that can be assigned to strings or primitives */
-                    if (attributeClass.isPrimitive() || String.class.isAssignableFrom(attributeClass)) {
-                        params.add(new QueryParameter()
-                                .type("string")
-                                .name("filter[" + typeName + "." + name + "][" + op.getNotation() + "]")
-                                .description("Filters the collection of " + typeName + " by the attribute "
-                                        + name + " " + "using the operator " + op.getNotation()));
-                    }
-                });
+            if (supportLegacyDialect) {
```

Comment:
I'm new to Elide, but would I be right in thinking that it only supports one dialect..  So for users that want to use the legacy dialect, then the RSQL examples should not be displayed.

Reply:
Elide supports multiple dialects enabled at the same time.